sidebars.js

```diff
@@ -151,45 +151,45 @@ module.exports = {
     },
     {
       type: 'category',
-      label: '🏭 lsp-factory.js',
+      label: '🖋️ eip191-signer.js',
       collapsed: true,
       items: [
         {
           type: 'autogenerated',
-          dirName: 'tools/lsp-factoryjs',
+          dirName: 'tools/eip191-signerjs',
         },
       ],
     },
     {
       type: 'category',
-      label: '🖋️ eip191-signer.js',
+      label: '📑 lsp-smart-contracts',
       collapsed: true,
       items: [
         {
           type: 'autogenerated',
-          dirName: 'tools/eip191-signerjs',
+          dirName: 'tools/lsp-smart-contracts',
         },
       ],
     },
     {
       type: 'category',
-      label: '📑 lsp-smart-contracts',
+      label: '📑 lsp-utils',
       collapsed: true,
       items: [
         {
           type: 'autogenerated',
-          dirName: 'tools/lsp-smart-contracts',
+          dirName: 'tools/lsp-utils',
         },
       ],
     },
     {
       type: 'category',
-      label: '📑 lsp-utils',
+      label: '🏭 lsp-factory.js',
```

Comment:
Maybe we should also mention deprecated in the sidebar menu?
Or put it under a submenu?

```suggestion
      label: '🏭 lsp-factory.js (deprecated)',
```